src/main/java/me/rayzr522/jsonmessage/JSONMessage.java

```diff
@@ -905,16 +1007,16 @@ public void setOnHover(MessageEvent onHover) {
         /**
          * @return The color
          */
-        public ChatColor getColor() {
+        public String getColor() {
             return color;
         }
 
         /**
          * @param color The color to set
          */
-        public void setColor(ChatColor color) {
-            if (!color.isColor()) {
-                throw new IllegalArgumentException(color.name() + " is not a color!");
+        public void setColor(String color) {
```

Comment:
same here as above, need to still have a `setColor(chatColor)` method (although much easier to maintain compatibility here)

Reply:
Added one and also annotated it with the deprecated annotation